doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -77,364 +77,403 @@ Complete the following steps to prepare the Thingy:91 for setup:
+      #. Scroll down in the menu on the left to :guilabel:`Device` and click :guilabel:`Write`.
 
-      .. note::
-         If you have issues updating modem firmware, click :guilabel:`Erase all` before trying to update the modem again. In this case, the contents of the flash memory are deleted and the applications must be reprogrammed.
+         .. figure:: /images/programmer_hex_write.png
+            :alt: Programmer - Writing of HEX files
 
-5. Update the nRF9160 SiP application:
+            Programmer - Writing of HEX files
 
-   a. Make sure the SWD selection switch **SW2** is set to **nRF91**.
+         The :guilabel:`MCUboot DFU` window appears.
 
-   #. In the menu on the left, click :guilabel:`Add file` in the File section, and select :guilabel:`Browse`.
-      A file explorer window appears.
+         .. figure:: /images/thingy91_mcuboot_dfu1.png
+            :alt: Programmer - MCUboot DFU
 
-      .. figure:: /images/programmer_add_file1.png
-         :alt: Programmer - Add file
+            Programmer - MCUboot DFU
 
-         Programmer - Add file
+      #. In the :guilabel:`MCUboot DFU` window, click :guilabel:`Write`.
+         When the update is complete, a :guilabel:`Completed successfully` message appears.
+      #. Scroll up in the menu on the left to :guilabel:`File` and click :guilabel:`Clear files`.
 
+   .. group-tab:: Through external debug probe
 
+      To program the application firmware using external debug probe, complete the following steps:
 
-   #. Navigate to the folder you downloaded and extracted from the `Nordic Semiconductor website`_ in the :ref:`Download firmware <download_firmware_thingy91>` step.
+      1. Open `nRF Connect for Desktop`_ and launch the Programmer application if you do not have it open already.
+      #. Make sure the SWD selection switch **SW2** is set to **nRF91**.
 
-   #. Open the folder that contains the HEX files for updating over USB.
-      See the :file:`CONTENTS.txt` file for information on which file you need.
+      #. In the menu on the left, click :guilabel:`Add file` in the File section, and select :guilabel:`Browse`.
+         A file explorer window appears.
 
-   #. Select the appropriate Asset Tracker v2 firmware file.
+         .. figure:: /images/programmer_add_file1.png
+            :alt: Programmer - Add file
 
-.. include:: ug_thingy91_gsg.rst
-   :start-after: programing_ext_dp_start
-   :end-before: programing_ext_dp_end
+            Programmer - Add file
+
+      #. Navigate to the folder you downloaded and extracted from the `Nordic Semiconductor website`_ in the :ref:`Download firmware <download_firmware_thingy91>` step.
+
+      #. Open the folder that contains the HEX files for updating over USB.
+         See the :file:`CONTENTS.txt` file for information on which file you need.
+
+      #. Select the appropriate Asset Tracker v2 firmware file.
+
+         .. include:: ug_thingy91_gsg.rst
```

Comment:
The include indents and numbers incorrectly:
![image](https://user-images.githubusercontent.com/83582261/174244904-231876d4-2a2b-49d4-8731-09dbdee36489.png)

Reply:
Yes, I think it messes up the numbering. The only other option seems to be duplicating the statements here as well.